src/env-inl.h

```diff
@@ -175,11 +188,17 @@ inline bool AsyncHooks::pop_async_id(double async_id) {
   async_id_fields_[kTriggerAsyncId] = async_ids_stack_[2 * offset + 1];
   fields_[kStackLength] = offset;
 
+  auto resources = execution_async_resources();
+  resources->Delete(env()->context(), offset).FromJust();
```

Comment:
It might be nice to stay consistent with using `.Check()`/`.FromJust()`.

I’d probably not use these at all here, as they aren’t guaranteed to succeed; using `USE(...);` instead seems reasonable to me.

Reply:
Fixed.